src/ComputationContainer/Share/Compare.cpp

```diff
@@ -55,8 +55,10 @@ bool operator<(const Share<FixedPoint> &left, const Share<FixedPoint> &right)
 // Docs/faster-comparison-operators.md
 bool operator==(const Share<FixedPoint> &left, const Share<FixedPoint> &right)
 {
-    auto x_ret = (left < right);
-    auto y_ret = (right < left);
+    // |x-y|<ep <=> (x-y<ep)&(y-x<ep)
+    auto epsilon = 0.0001;
+    auto x_ret = (left < right + epsilon);
+    auto y_ret = (right < left + epsilon);
```

Comment:
If you add ε, won't the following expression always be `0` when `==` operation does with same value?

https://github.com/acompany-develop/QuickMPC/blob/b220899ec073f6be9f67cc27938240968e8223dc/src/ComputationContainer/Share/Compare.cpp#L146

Reply:
resolve it